meta-refkit-core/bbappends/meta-security/recipes-tpm/swtpm/swtpm-wrappers-native.bbappend

```diff
@@ -0,0 +1,4 @@
+# Temporary workaround (needs to be fixed in meta-security once the
+# necessary patch "net-tools: enable native and nativesdk variant"
+# is in OE-core): swtpm_setup.sh needs netstat command.
```

Comment:
Appears to be in oe-core (https://github.com/openembedded/openembedded-core/commit/508163bef169cf0d9aa97e73c02d1ecc68480e91). Can this be removed?